src/ContentHubClient.php

```diff
@@ -1281,6 +1281,29 @@ public function startScrollByFilter(string $filter_uuid, string $scroll_time_win
     ]));
   }
 
+  /**
+   * Initiates Scroll API request chain.
+   *
+   * @param string $scroll_time_window
+   *   How long the scroll cursor will be retained inside memory. Must be
+   *   suffixed with duration unit (m, s, ms etc.).
+   * @param int $size
+   *   Amount of entities to return.
+   *
+   * @return array
+   *   Response from scroll API.
+   *
+   * @throws \Exception
+   */
+  public function startScroll(string $scroll_time_window = '30m', int $size = 100): array {
```

Comment:
For a complete implementation we'd need to have the query part specified as well. The `/scroll` endpoint accepts an ElasticSearch compatible query under the `query ` key.

Reply:
Just like in case of `/_search` endpoint.